rw-system.sh

```diff
@@ -309,6 +335,10 @@ if getprop ro.vendor.build.fingerprint | grep -qE '^xiaomi/(daisy|wayne)/(daisy|
     setprop audio.camerasound.force true
 fi
 
+if getprop ro.boot.fpsensor | grep -qF goodix; then
```

Comment:
I really want a whitelist of device names there

Reply:
I can do a whitelist for only Xiaomi devices if that's better.